whatToDad_site/whatToDad_app/templates/activity.html

```diff
@@ -26,9 +26,52 @@ <h1> Activities </h1>
     <br>
     <form method="POST" action="{% url 'activity' %}">
         {% csrf_token %}
-        {{ form }}
-        <input type="button" class="btn btn-secondary" value="Cancel" data-dismiss="modal"></input>
-        <input type="submit" class="btn btn-primary" name="add" value="Add activity"></input>
+        <div class="row">
+            <div class="col">
+                <h3 class="form-control">Detail: {{ form.detail }}</h3>
+            </div>
+
+            <div class="col">
+                <h3 class="form-control">Min-age: {{ form.min_age }}</h3>
+            </div>
+
+            <div class="col">
+                <h3 class="form-control">Max-age: {{ form.max_age }}</h3>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col">
+                <h3 class="form-control">AddressNumber: {{ form.addressNumber }}</h3>
```

Comment:
While these are numbers, they aren't numbers in any semantic sense. That is, most of the properties like addition and subtraction or greater-than and less-than  have no role to play for these. On mobile devices number fields will hint that the keyboard shown use a number pad which is helpful, but the increment/decrement buttons in the input form on desktop are confusing and unhelpful.

It also restricts users with odd addresses (even in the US). Addresses are hard and very difficult to get right. 

https://www.qalocate.com/addresses-are-complicated/

**Feel free to not adjust the code to this comment**. I just wanted to share a whole rabbit hole of misery you go down when you start trying to store and work with addresses, some of which I've faced myself.